src/App.tsx

```diff
@@ -15,21 +15,23 @@ const HomePage = lazy(() => import('./pages/HomePage'));
 const Admin = lazy(() => import('./pages/admin/index'));
 const ImportVideo = lazy(() => import('./pages/admin/import-video'));
 const CreateBlog = lazy(() => import('./pages/admin/create-blog'));
-
+const CreateNotes = lazy(() => import('./pages/admin/create-notes'));
 
 class App extends React.Component<Props, State>  {
   render() {
     return (
       <Suspense fallback={<div></div>}>
         <Switch key={this.props.location.pathname}>
-          <Route exact path="/" render={props => <HomePage isVideo="false" isBlog="false" {...props} />} />
+          <Route exact path="/" render={props => <HomePage isVideo="false" isBlog="false" isNotes="false" {...props} />} />
```

Comment:
Since these `isXXX` variables are all mutually exclusive you may want to switch to a pattern like [this blog](https://spicefactory.co/blog/2019/03/26/how-to-avoid-the-boolean-trap-when-designing-react-components/)

Reply:
I'll have to save this one for later, but agreed.